test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/BuildCommand.cs

```diff
@@ -199,6 +200,16 @@ private string NoDependencies
             }
         }
 
+        private string Verbose
+        {
+            get
+            {
+                return _verbose ?
+                    "--verbose" :
+                    "";
```

Comment:
We should make the operators first on the line of the operand instead of last on the preceding line.